requirements_dev.txt

```diff
@@ -6,4 +6,6 @@ mkdocs-material~=8.2.8
 pep8-naming~=0.12.1
 pytest-runner~=6.0.0
 pytest~=7.1.1
-recommonmark~=0.7.1
\ No newline at end of file
+recommonmark~=0.7.1
+pytest-postgresql==4.1.1
+psycopg==3.0.16
```

Comment:
Are these really dev dependencies if the fixture is intended to be used in other apps? These will probably not be installed transitively.